tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "types": ["node", "@wdio/globals/types", "@wdio/mocha-framework"],
+    "allowJs": true,
```

Comment:
Do you mind aligning this one with the [**tsconfig.json** in **forms-manager**](https://github.com/DEFRA/forms-manager/blob/main/tsconfig.json) too?

Mainly we add `"checkJs": true,` and `"strict": true,` to check for mistakes

Reply:
done